docs/definition-language.md

```diff
@@ -178,11 +178,11 @@ There are a set of options to control the model definition.
 
 - strict:
     - true: Only properties defined in the model are accepted. Use this
-    mode if you want to make sure only predefined properties are accepted.
+    mode if you want to make sure only predefined properties are accepted. Relational databases only support this setting.
```

Comment:
> Relational databases only support this setting

Only relational databases support this setting.

Reply:
NoSQL DBs support this setting as well.